sysdig/resource_sysdig_monitor_dashboard.go

```diff
@@ -53,6 +53,35 @@ func resourceSysdigMonitorDashboard() *schema.Resource {
 				ComputedWhen: []string{"public"},
 				Computed:     true,
 			},
+			"share": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"member": {
+							Type:     schema.TypeSet,
+							MaxItems: 1,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"type": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"id": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+								},
+							},
+						},
+						"role": {
```

Comment:
Similar to above, I only see `ROLE_RESOURCE_READ` and `ROLE_RESOURCE_EDIT` used in the Python client. If there is a definitive list of what's accepted I can update the docs accordingly.